### DISCUSSION TITLE
caption-side in Tailwinds?

### DISCUSSION THREAD

**Discussion by MattAppleton** [tailwindlabs/tailwindcss#3759] [Ideas]
I am building a layout where I want the text elements of my 'tile' / card (which have _variable heights + widths_ determined using packery.js)  to never go wider than the image.

So far my only solution css-wise seems to be using figure + figcaption like this...

Any thoughts on whether 'caption-side' properties might be a valuable addition to Tailwind? Maybe there's another way to solve this but hadn't managed to find it with various flex box fiddles and the like.

```
<div class="max-w-full ">

    <div class="border-8  border-white">
        <figure class="table">

            <figcaption class="table-caption" style="caption-side: bottom">

                my caption in here...

            </figcaption>

            <img src="{{ image.cropResize(700, 700).url }}" >

        </figure>

    </div>

</div>

```

**Comment by MichaelAllenWarner** [comment]
I agree that adding [`caption-side`](https://developer.mozilla.org/en-US/docs/Web/CSS/caption-side) utilities to core is a good idea. I use precisely the same `figure.table`/`figcaption.table-caption` pattern mentioned above in every project, and I use a custom `caption-bottom` utility class to make it work.
